packages/@uppy/transloadit/src/Assembly.ts

```diff
@@ -101,16 +101,30 @@ class TransloaditAssembly extends Emitter {
     })
 
     this.#sse.addEventListener('assembly_upload_finished', (e) => {
-      const file = JSON.parse(e.data)
-      this.status.uploads.push(file)
+      const file = JSON.parse(e.data) as AssemblyFile
+      let uploads = this.status.uploads
+      if (!uploads) {
+        uploads = [] as NonNullable<AssemblyResponse['uploads']>
+        this.status.uploads = uploads
+      }
```

Comment:
Although I even wonder if it's needed at all, it wasn't before
```suggestion
      this.status.uploads ??= []
```